.github/workflows/ci.yml

```diff
@@ -115,10 +113,33 @@ jobs:
     - name: Build
       shell: bash -l {0}
       run: |
-        CMAKE_OPTIONS="-DHIGHFIVE_USE_XTENSOR:BOOL=ON"
+        CMAKE_OPTIONS=(-DHIGHFIVE_USE_XTENSOR:BOOL=ON)
         source $GITHUB_WORKSPACE/.github/build.sh
 
     - name: Test
       working-directory: ${{github.workspace}}/build
       shell: bash -l {0}
       run: ctest --output-on-failure -C $BUILD_TYPE
+
+  # Job producing code coverage report
```

Comment:
I'd move the job in its own file given GHA experience. Fine with me either way

Reply:
we can do this at some point when the pipeline grows to clean up a bit. I'm gonna merge as is.